src/main/java/dev/waterdog/waterdogpe/ProxyServer.java

```diff
@@ -298,20 +311,7 @@ public boolean dispatchCommand(CommandSender sender, String message) {
             return false;
         }
 
-        Command command = this.getCommandMap().getCommand(args[0]);
```

Comment:
Looks like this reverts some of the recent changes on master.

Reply:
Fixed in [8a058ff](https://github.com/WaterdogPE/WaterdogPE/pull/216/commits/8a058ff55bab2328119d76d8b59761bb9b30c853)